engine/src/main/java/org/terasology/rendering/nui/animation/SwipeMenuAnimationSystem.java

```diff
@@ -181,10 +181,14 @@ public void update(float delta) {
             // avoid skipping over fast animations on heavy load
             animDelta = 0.1f;
         }
+        if(animDelta == 0.0f) {
```

Comment:
Trivial code style issue: space between `if` and `(` - also may help with a newline between the two blocks.

Quirky float risk: comparisons can get weird with them, maybe do `animDelta < 0.0001f` instead?